addons/account_edi_ubl/models/account_edi_format.py

```diff
@@ -202,12 +203,12 @@ def format_monetary(amount):
     def _export_ubl(self, invoice):
         self.ensure_one()
         # Create file content.
-        xml_content = b"<?xml version='1.0' encoding='UTF-8'?>"
+        xml_content = markupsafe.Markup("<?xml version='1.0' encoding='UTF-8'?>")
```

Comment:
useful? the result is stored in an attachment, and will be retrieved as b64 or bytes, so whatever Markup is generated here will be discarded immediately, no?